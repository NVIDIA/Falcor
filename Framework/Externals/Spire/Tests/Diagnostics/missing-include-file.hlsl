// trying to include a non-existant file

#include "does-not-exist.h"
