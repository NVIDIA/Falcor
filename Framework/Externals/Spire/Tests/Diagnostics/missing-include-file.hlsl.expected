result code = -1
standard error = {
Tests/Diagnostics/missing-include-file.hlsl(3): error 15300: failed to find include file 'does-not-exist.h'
}
standard output = {
}
